Added workaround - 2.1x to 2.2x upgrade Minio failure (#4643)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

    kubectl delete pod -n $WORKSPACE_NAMESPACE $(kubectl get pod -l app.kubernetes.io/name=$SPARK_OPERATOR_RELEASE_NAME -n $WORKSPACE_NAMESPACE -o jsonpath='{range .items[0]}{.metadata.name}')
    ```

### Minio Disk insufficient space when upgrading

> If upgrading DKP version 2.1.x to 2.2.x, there is a bug that does not allow upgrading due to insufficient space on the MinIO Disk. To address this issue, disable `fluent-bit` in the `AppDeployments` before upgrading.

Reviewer: ```suggestion
When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue,  we recommend that you disable the `fluent-bit` Platform Application before upgrading.   
```

> When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue,  we recommend that you disable the `fluent-bit` Platform Application before upgrading.   

Author: ```suggestion
When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue,  we recommend that you disable the `fluent-bit` Platform Application before upgrading.   
```

> When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue,  we recommend that you disable the `fluent-bit` Platform Application before upgrading.   

Author: ```suggestion
When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, we recommend that you disable the `fluent-bit` Platform Application before upgrading. 
```
